discourse.tf

```diff
@@ -16,4 +19,7 @@ module "discourse-staging" {
     discourse_elasticache_instance_size = "cache.t2.micro"
     service_security_group_id           = "${module.mesos-cluster-staging.mesos-cluster-slave-sg-id}"
     elasticache_subnet_group            = "${aws_elasticache_subnet_group.elasticache-staging-subnet-group.name}"
+    fqdn                                = "discourse.staging.paas.mozilla.community"
+    ssl_certificates                    = "${var.ssl_certificates}"
```

Comment:
You can always pass the specific certificate here instead of passing everything to the module.